### DISCUSSION TITLE
Can not able to remove "Restart login" text/ link by using custom theme

### DISCUSSION THREAD

**Discussion by namita-25** [keycloak/keycloak#21435] [Q&A]
-I have added custom theme folder in project directory of react js
-I have deleted "Restart login" text from login-password.ftl file
-Still I am able to see "Restart login" text in output.
-please find attached screen shot 

![Screenshot from 2023-07-04 17-54-42](https://github.com/keycloak/keycloak/assets/112607821/fb9c5b78-86ac-4a26-89b5-c2552fe4e618)

**Comment by jonkoops** [comment]
I'd recommend you read ["How do I ask a good question?"](https://stackoverflow.com/help/how-to-ask), there is not a lot of information to go on in your question.
